tpl/inc/modal.header.php

```diff
@@ -15,12 +15,23 @@
 
 <link rel="stylesheet" href="<?php echo esc_url( LSWCP_PLUGIN_URL ); ?>assets/css/litespeed.css" />
 
-<div class="litespeed litespeed-modal">
+<div class="litespeed litespeed-modal<?php 
+if(isset($_progress) && $_progress ) { 
+    echo ' litespeed-progress';
+} 
+?>"<?php if($_id ) { echo ' id="' . $_id . '"'; 
+}  ?>>
```

Comment:
WordPress core lack a template engine.
I would introduce a very tiny engine to build HTML.
e.g. https://github.com/szepeviktor/SentencePress/tree/master/src/Html
( `?>"<?` looks like ASCII art)

Reply:
```php
$modal_attrs = [];
$modal_classes = [
    'litespeed',
    'litespeed-modal',
];

if (isset($_progress) && $_progress) {
    $modal_classes[] = 'litespeed-progress';
}.
if ($_id) {
    $modal_attrs['id'] = $_id;
}

$modal_attrs['class'] = implode(' ', $modal_classes);
$modal = new SzepeViktor\SentencePress\Html\Element(
    'div',
    $modal_attrs,
    'The Content'
);

print $modal->render();
```